src/python/other-bots/chromium-tests-syncer/run.py

```diff
@@ -85,7 +89,9 @@ def unpack_crash_testcases(crash_testcases_directory):
 
     # 8. Skip in-process fuzzer testcases, since these are only applicable to
     # fuzz targets and don't run with blackbox binaries.
-    if testcase.fuzzer_name and testcase.fuzzer_name in ['afl', 'libFuzzer']:
+    if testcase.fuzzer_name and testcase.fuzzer_name in [
+        'afl', 'centipede', 'googlefuzztest', 'libFuzzer'
```

Comment:
Maybe define a constant for this at the top, e.g. `ENGINE_FUZZER_NAMES`.

Reply:
Done.